src/content/docs/apm/agents/net-agent/net-agent-api/net-agent-api.mdx

```diff
@@ -2334,7 +2334,7 @@ The following list contains the different calls you can make with the API, inclu
 
     ### Description
 
-    Sets the name of the current transaction. Before you use this call, ensure you understand the implications of [metric grouping issues](/docs/agents/manage-apm-agents/troubleshooting/metric-grouping-issues).
+    Sets the name (other than the initial `WebTransaction` or `OtherTransaction` base name) of the current transaction. Before you use this call, ensure you understand the implications of [metric grouping issues](/docs/agents/manage-apm-agents/troubleshooting/metric-grouping-issues).
```

Comment:
I feel like this needs to be re-worded a bit -- maybe to say something like "Sets a custom transaction name (appearing after the `WebTransaction` or `OtherTransaction` prefix) for the current transaction. ..." ?

The example code makes it very clear, but the words here don't.

Reply:
Thanks for the feedback.  I wasn't super happy with my initial wording, and I like yours better.  I pushed a change.